.github/workflows/auto-approve.yml

```
@@ -0,0 +1,17 @@
+# ~~ Generated by projen. To modify, edit .projenrc.js and run "npx projen".
+
+name: auto-approve
+on:
+  pull_request:
+    types:
+      - labeled
+      - opened
+      - unlabeled
+jobs:
+  approve:
+    if: contains(github.event.pull_request.labels.*.name, 'pr/auto-approve')
```

I don't see a huge value in the `pr/` prefix here.

```suggestion
    if: contains(github.event.pull_request.labels.*.name, 'auto-approve')
```